singlecellexperiment-create-single-cell-experiment.R

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env Rscript 
+
+# Load optparse we need to check inputs
+
+suppressPackageStartupMessages(require(optparse))
+
+# Source common functions
+
+ca <- commandArgs()
+script_dir <- dirname(sub('--file=', '', ca[grep('--file', ca)]))
```

Comment:
I'm not a big fan of this pattern of sourcing, I understand of course that the alternative is a bit burdensome (have an R package for these scripts).